src/DotLiquid/StandardFilters.cs

```diff
@@ -118,6 +118,11 @@ public static string Capitalize(string input)
             if (input.IsNullOrWhiteSpace())
                 return input;
 
+            if (Template.SyntaxCompatibilityLevel >= SyntaxCompatibility.COMPAT_2_1) {
+                string trimmed = input.TrimStart();

```

Comment:
nit: consider using var
I noticed at this time the code in this repo is mixing implicit and explicit types, not sure it is following a strict rule for it.